### ISSUE TITLE
Can't use spliceSlot on arrayfield with m-f-v

### ISSUE THREAD

**Issue by alexrussell** [boughtbymany/mutt-forms-vue#52]
When you call `spliceSlot` on an array field, it eventually tries to call `destroy` on the widget (and the field's sub-widgets if an object field). At a guess, that's [here](https://github.com/boughtbymany/mutt-forms/blob/b470ca74fbbb1b4e926161146e55c5fdbc0db60b/src/widgets/core.js#L82) but for whatever reason (at a guess due to m-f-v taking over the actual DOM rendering) there is no element returned by `this.getElementWrapper()`, so the next line errors out.

This is currently stopping me from implementing a "remove pet" feature in ED.

**Comment by alexrussell** [comment]
Wasn't sure if this was mutt-forms or mutt-forms-vue, but being as the problem occurs when using mutt-forms-vuew, I figured I'd put it here. It's possible mutt-forms will have to actually include the fix, however.

**Comment by alexrussell** [comment]
Ahh I have managed to bypass the logic that does this by passing false to the second parameter of `spliceSlot` (`updateWidget`). I feel that this is still something of a hack and that m-f-v should either proxy `spliceSlot` and silently pass that `false` in for me, or it should otherwise be addressed, so I'll leave the issue open, but obviously you know the lib better than I do, so by all means close.

**Comment by nicksnell** [comment]
@alexrussell yeah you are right, it's not ideal right now. The is a bit of work underway to decouple the HTML widgets from Mutt, so mutt doesn't have this sort of problems when used with a different interface - like mfv.

Will leave open for now so we remember to address this specifically.
